site/src/api/api.ts

```diff
@@ -311,8 +311,8 @@ export const updateUserPassword = async (
   updatePassword: TypesGen.UpdateUserPasswordRequest,
 ): Promise<undefined> => axios.put(`/api/v2/users/${userId}/password`, updatePassword)
 
-export const getSiteRoles = async (): Promise<Array<TypesGen.Role>> => {
-  const response = await axios.get<Array<TypesGen.Role>>(`/api/v2/users/roles`)
+export const getSiteRoles = async (): Promise<Array<TypesGen.AssignableRoles>> => {
+  const response = await axios.get<Array<TypesGen.AssignableRoles>>(`/api/v2/users/roles`)
```

Comment:
Eventually, it would be nice differentiate the two `roles` endpoints by label a bit more. In the network tab, all I see is "roles" and it's hard to filter or figure out what I'm looking at.

Reply:
Not a bad idea